### DISCUSSION TITLE
Netspace is dropping from ATH ~3200PiB to ~3000PiB

### DISCUSSION THREAD

**Discussion by zjshen14** [Chia-Network/chia-blockchain#4393] [Q&A]
According to the [explorer](https://www.chiaexplorer.com/charts/netspace), any idea on what happened today?

**Comment by mariano54** [comment]
Everyone should update to 1.1.5, because 1.1.4 and older might be making invalid blocks that are ignored by peers. This is probably why we see a lower netspace, many people have not updated.

**Reply by SackofFluor** [reply]
but so fat that would mean that so far around 20-30% are invalid blocks?!
That would be massive!

How can I check whether mine are valid?

**Reply by zjshen14** [reply]
@mariano54 thx for the response.

One question, is making invalid blocks also affecting block syncing? If not, the farmers could still be synced, and farm at with correct netspace?

Also, apart from the 1.1.5 release notes, I didn't see the upgrade notification from official communication channel. That's probably why many people haven't upgraded. I recommended this be communicated sooner than later.

**Reply by keeperlink** [reply]
As I understand the bug doesn't affect the harvester nodes, as long as the master node updated, right?

**Comment by zjshen14** [comment]
BTW, it continuously drops to 2.75 EiB according to https://www.chiaexplorer.com/charts/netspace
